src/Ext/Techno/Body.h

```diff
@@ -34,15 +35,26 @@ class TechnoExt
 		AnimTypeClass* MindControlRingAnimType;
 		OptionalStruct<int, false> DamageNumberOffset;
 		bool IsInTunnel;
+		bool IsBurrowed;
 		bool HasBeenPlacedOnMap; // Set to true on first Unlimbo() call.
 		CDTimerClass DeployFireTimer;
 		bool ForceFullRearmDelay;
 		int WHAnimRemainingCreationInterval;
+		std::vector<std::unique_ptr<AttachEffectClass>> AttachedEffects;
```

Comment:
```suggestion
		std::vector<AttachEffectClass> AttachedEffects;
```
[a container of instances](https://github.com/Ares-Developers/Ares/blob/master/src/Ext/Techno/Body.h#L64) of instances should suffice (

Reply:
Not implementing for now due to amount of code refactoring required beyond the suggested change.